### PR TITLE
Made some format changes. Added new rule to requests and ideas.

### PR DESCRIPTION
**Pull request by Darkhax**
Proposing a new rule to the Requests and Ideas section. This rule will require posts in the Request/Ideas section to actually contain the idea for the mod. Posts saying "I have an idea and I want someone to code it for me, pm me and I will then tell you what the idea is" seems like it would be better suited for a "team" request. 

I also fixed the wording of the "no picks, no clicks" and added a few periods.

### CODE REVIEW

File: rules/mapping_and_modding/rules.markdown [hunk 1]
```diff
@@ -29,6 +29,7 @@ report the topic using the Report feature and request it to be moved.
 ### Requests / Ideas For Mods.
 
 * Topics must be either a request or an idea for a mod
+* Topics must state the idea for the mod in the post.
```
Comment (LordRalex) [inline rules/mapping_and_modding/rules.markdown]: What is this saying? It is requests or ideas, what purpose is this doing

File: rules/mapping_and_modding/rules.markdown [hunk 2]
```diff
@@ -3,18 +3,18 @@
 ## Section Rules
 
 * No linking to sites that require a person to do surveys or require personal information, such as cell phone numbers.
-* Do not ask or pressure content to be updated
+* Do not ask or pressure content to be updated.
```
Comment (LordRalex) [inline rules/mapping_and_modding/rules.markdown]: Why the full stops

File: rules/mapping_and_modding/rules.markdown [hunk 3]
```diff
@@ -3,18 +3,18 @@
...
 * Topics that offer downloadable content must follow the download links rules, located here: [Download Link Rules](#mapping-and-modding:download-links)
 * Topics that offer downloadable content must follow the file guidelines, located here: [File Guidelines](#mapping-and-modding:file-guidelines) 
 * Topics that offer downloadable content must follow the tagging format, which can be found here: [Tagging Format](#mapping-and-modding:tagging-format)
-* Topics must have an installation guide if required
-* Any required permissions must be displayed in the original post
+* Topics must have an installation guide if required.
```
Comment (LordRalex) [inline rules/mapping_and_modding/rules.markdown]: Same here, what is going on with the mixed full stops?

**Comment by LordRalex** [comment]
The full stops are going against the formatting of the rules
`List items should not end with a full stop (unless it's the paragraph description of the item)`

As those are lists, they should not end with it

File: rules/mapping_and_modding/rules.markdown [hunk 4]
```diff
@@ -3,18 +3,18 @@
...
+* Any required permissions must be displayed in the original post.
 * Topics or posts made to start or continue a flame war against another mod, a user, or a group, is not allowed.
-* "No pics, no clicks" are considered spam, and are not allowed
+* Posts saying "No pics, no clicks" are considered spam and are not allowed.
```
Comment (d3fin3d) [inline rules/mapping_and_modding/rules.markdown]: "containing" would read better than "saying".

**Comment by Darkhax** [comment]
@LordRalex I didn't catch the Full Stops bit the first time around. I noticed some of them had it while others didn't so I thought I would match them. That was obviously a mistake. 

As for what that new rule would do. Often times people make posts in the request/ideas section saying something along the lines of "I have an idea for a new mod, pm me if you want to make it for me". I feel that posts in this section should clearly contain the idea they want. If you're going to request a mod be made you should actually state information about that idea in the topic rather than requesting people who are interested pm you before you give them any information on the project.

**Comment by Darkhax** [comment]
@d3fin3d I agree. Contains would be a better word for this.

**Comment by LordRalex** [comment]
@darkhax That was not clear when you put it there then. You need to make that a bit cleaner so that anyone can read it and know what it means.

File: rules/mapping_and_modding/rules.markdown [hunk 5]
```diff
@@ -9,8 +9,8 @@
 * Topics that offer downloadable content must follow the tagging format, which can be found here: [Tagging Format](#mapping-and-modding:tagging-format)
 * Topics must have an installation guide if required
 * Any required permissions must be displayed in the original post
-* Topics or posts made to start or continue a flame war against another mod, a user, or a group, is not allowed.
-* "No pics, no clicks" are considered spam, and are not allowed
+* Topics or posts made to start or continue a flame war against another mod, a user, or a group, are not allowed
+* Posts containing "No pics, no clicks" are considered spam and are not allowed
```
Comment (LordRalex) [inline rules/mapping_and_modding/rules.markdown]: I keep reading it, and this just makes it sound like you cannot say that phrase at all. It is only bad when it is by itself. Posts that have it but explain what to do to fix it should not be considered spam, which this is implying they would be.

**Pull request merged**